src/luascript.cpp

```diff
@@ -3991,7 +3991,15 @@ int LuaScriptInterface::luaGameLoadMap(lua_State* L)
 {
 	// Game.loadMap(path)
 	const std::string& path = getString(L, 1);
-	g_dispatcher.addTask(createTask(std::bind(&Game::loadMap, &g_game, path)));
+	g_dispatcher.addTask(createTask( [path]() {
```

Comment:
We don't usually put spaces between ( and [, and I think `path` should be passed by reference.

Reply:
You can't pass path by reference, because the temporary returned from getString() could expire before the lambda is executed.

Reply:
Sure, but you could move it then

Reply:
Is move capture allowed in C++11?

Reply:
One more to #2396